test/fixtures/auth/githubUserInfo.js

```diff
@@ -101,5 +101,49 @@ module.exports = () => {
+      provider: "github",
+      _raw: '{"login":"ravikumar1002","id":86847625,"node_id":"MDQ6VXNlcjg2ODQ3NjI1","avatar_url":"https://avatars.githubusercontent.com/u/86847625?v=4","gravatar_id":"","url":"https://api.github.com/users/ravikumar1002","html_url":"https://github.com/ravikumar1002","followers_url":"https://api.github.com/users/ravikumar1002/followers","following_url":"https://api.github.com/users/ravikumar1002/following{/other_user}","gists_url":"https://api.github.com/users/ravikumar1002/gists{/gist_id}","starred_url":"https://api.github.com/users/ravikumar1002/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/ravikumar1002/subscriptions","organizations_url":"https://api.github.com/users/ravikumar1002/orgs","repos_url":"https://api.github.com/users/ravikumar1002/repos","events_url":"https://api.github.com/users/ravikumar1002/events{/privacy}","received_events_url":"https://api.github.com/users/ravikumar1002/received_events","type":"User","site_admin":false,"name":"Ravi kumar","company":null,"blog":"https://ravikumar.dev/","location":"Delhi, India","email":null,"hireable":null,"bio":"Frontend developer","twitter_username":"kumarravi1002","public_repos":47,"public_gists":0,"followers":9,"following":13,"created_at":"2021-07-02T16:40:44Z","updated_at":"2023-07-27T00:12:32Z"}',
+      _json: {
+        login: "ravikumar1002",
+        id: 86847625,
+        node_id: "MDQ6VXNlcjg2ODQ3NjI1",
+        avatar_url: "https://avatars.githubusercontent.com/u/86847625?v=4",
+        gravatar_id: "",
+        url: "https://api.github.com/users/ravikumar1002",
+        html_url: "https://github.com/ravikumar1002",
+        followers_url: "https://api.github.com/users/ravikumar1002/followers",
+        following_url: "https://api.github.com/users/ravikumar1002/following{/other_user}",
+        gists_url: "https://api.github.com/users/ravikumar1002/gists{/gist_id}",
+        starred_url: "https://api.github.com/users/ravikumar1002/starred{/owner}{/repo}",
+        subscriptions_url: "https://api.github.com/users/ravikumar1002/subscriptions",
+        organizations_url: "https://api.github.com/users/ravikumar1002/orgs",
+        repos_url: "https://api.github.com/users/ravikumar1002/repos",
+        events_url: "https://api.github.com/users/ravikumar1002/events{/privacy}",
+        received_events_url: "https://api.github.com/users/ravikumar1002/received_events",
+        type: "User",
+        site_admin: false,
+        name: "Ravi kumar",
+        company: null,
+        blog: "https://ravikumar.dev/",
+        location: "Delhi, India",
+        email: null,
+        hireable: null,
+        bio: "Frontend developer",
+        twitter_username: "kumarravi1002",
+        public_repos: 47,
+        public_gists: 0,
+        followers: 9,
+        following: 13,
+        created_at: "2021-07-02T16:40:44Z",
+        updated_at: "2023-07-27T00:12:32Z",
+      },
+    },
   ];
```

Comment:
Here no need to create one more fixture 
add to previous data itself

Reply:
ok